data/input_files/run_serac_tests.sh

```diff
@@ -0,0 +1,105 @@
+#! /bin/bash
+# desc: runs all inputs on serac moves their summary.json files
+# to the tests integration directory
+# notes:
+#  - this is meant to run in ./serac_repo/data/input_files
+#  - SERAC_EXE can be found in a build directory under "bin"
+#  - you can run without sbatch with IS_PARALLEL=0
+#  - KEEP_OUTPUTFILE for debugging
+
+#SBATCH -J run_serac_tests
+#SBATCH -t 5:00:00
+#SBATCH -p pbatch
+#SBATCH -N 1
+#SBATCH -n 2
+
+if [ $# -ne 3 ]
+    then
+    echo "args: <SERAC_EXE> <IS_PARALLEL> <KEEP_OUTPUTFILE>"
+    exit 1
+fi
+
+SERAC_EXE=$1
+IS_PARALLEL=$2
+KEEP_OUTPUTFILE=$3
+
+# set dir vars
+INPUT_DIR=$(pwd)
+OUTPUT_DIR=$(pwd)/../output_files
+SERAC_DIR="${INPUT_DIR}/../../"
+INTEGRATION_DIR="${SERAC_DIR}/tests/integration"
+cd ${INPUT_DIR}
+
+# create an error log, keeping track of inputs that fail
+echo "Error with..." > ${INPUT_DIR}/error.log 
+
+# if output directory exists, delete it
+if [ ! -f ${OUTPUT_DIR} ]
+    then
+    rm -rf ${OUTPUT_DIR}
+fi
+
+# create output directory
+mkdir -p ${OUTPUT_DIR}
+
+for f in ./*/*/*.lua
```

Comment:
You should use globstar here in case the directory structure changes.  I realize we only have lua files at a certain level but that may change.  Here is a page that explains it:

https://www.linuxjournal.com/content/globstar-new-bash-globbing-option

Reply:
i figured there was a better way to do this. the only issue is that if the files are organized differently later on will also cause problems elsewhere.

for example i am getting the category (like "solid") and resulting file ("dyn_direct_solve") using the file structure.
with `tests/solid/dyn_direct_solve.lua`:
`CATEGORY=$(echo $f | awk -F "/" '{print $3}')`
`RES_FILE=$(echo $f | awk -F "/" '{print $4}' | awk -F "." '{print $1}')`

Reply:
True.. we can always fix this down the road if/when we hit it.